patches/@react-native+gradle-plugin+0.72.11.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/ReactPlugin.kt b/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/ReactPlugin.kt
+index 062dcc7..cb400ec 100644
+--- a/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/ReactPlugin.kt
++++ b/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/ReactPlugin.kt
+@@ -15,6 +15,7 @@ import com.facebook.react.tasks.GenerateCodegenArtifactsTask
+ import com.facebook.react.tasks.GenerateCodegenSchemaTask
+ import com.facebook.react.utils.AgpConfiguratorUtils.configureBuildConfigFields
+ import com.facebook.react.utils.AgpConfiguratorUtils.configureDevPorts
++import com.facebook.react.utils.AgpConfiguratorUtils.configureNamespaceForLibraries
+ import com.facebook.react.utils.BackwardCompatUtils.configureBackwardCompatibilityReactMap
+ import com.facebook.react.utils.DependencyUtils.configureDependencies
+ import com.facebook.react.utils.DependencyUtils.configureRepositories
+@@ -64,6 +65,7 @@ class ReactPlugin : Plugin<Project> {
+ 
+       configureReactNativeNdk(project, extension)
+       configureBuildConfigFields(project)
++      configureNamespaceForLibraries(project)
+       configureDevPorts(project)
+       configureBackwardCompatibilityReactMap(project)
+ 
+diff --git a/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt b/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt
+index 3bf0085..59397b4 100644
+--- a/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt
++++ b/node_modules/@react-native/gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt
+@@ -8,11 +8,17 @@
+ package com.facebook.react.utils
+ 
+ import com.android.build.api.variant.AndroidComponentsExtension
++import com.facebook.react.ReactExtension
+ import com.facebook.react.utils.ProjectUtils.isHermesEnabled
+ import com.facebook.react.utils.ProjectUtils.isNewArchEnabled
+ import org.gradle.api.Action
+ import org.gradle.api.Project
+ import org.gradle.api.plugins.AppliedPlugin
++import java.io.File
++import com.android.build.gradle.LibraryExtension
++import javax.xml.parsers.DocumentBuilder
++import javax.xml.parsers.DocumentBuilderFactory
++import org.w3c.dom.Element
+ 
+ @Suppress("UnstableApiUsage")
+ internal object AgpConfiguratorUtils {
+@@ -49,6 +55,40 @@ internal object AgpConfiguratorUtils {
+     project.pluginManager.withPlugin("com.android.application", action)
+     project.pluginManager.withPlugin("com.android.library", action)
+   }
++
++  fun configureNamespaceForLibraries(appProject: Project) {
```

Comment:
Why is it needed?

Reply:
Would be nice to add some comment

Reply:
Personally I have not changed this line - @mczernek Maybe you remember what was the reason behind this change?

Reply:
Because, Gradle from some version requires each and every library to declare namespace, and since not all of them do, we need to do that for them.

Here's more info about that: https://github.com/react-native-community/discussions-and-proposals/issues/671